src/utils/command_utils.ts

```diff
@@ -36,19 +37,87 @@ export class CommandUtility {
         this.receiptHandler = new ReceiptHandler(this.persistence, this.read.getPersistenceReader(), this.modify)
     }
 
+    private async getAppUser(): Promise<IUser | undefined> {
```

Comment:
Improved layering, and better definition of the "schema", applicable to all user-centric apps.